### PR TITLE
changes in contributing guidelines

### PR DESCRIPTION
**Pull request by kimerran**
getting things in order. let's finalize our workflow.

### CODE REVIEW

**Review by ac-opensource — APPROVED** [review]

**Review by TheoryOfNekomata — COMMENTED** [review]
Is Item 5 in work-flow intended to be omitted?

**Comment by kimerran** [comment]
@Temoto-kun yes. it has been removed since we now require at least 2 approvals before we proceed to merge any PRs

**Review by ghost — APPROVED** [review]

**Pull request merged**
